src/inspector/node_protocol.pdl

```diff
@@ -92,3 +92,15 @@ experimental domain NodeWorker
       # Identifier of a session which sends a message.
       SessionID sessionId
       string message
+
+# Support for inspecting node process state.
+experimental domain NodeRuntime
+  # When domain is enabled `NodeRuntime.waitingForDisconnect` will be issued at the end of node process.
+  # Please see `NodeRuntime.waitingForDisconnect` for more details.
+  command enable
+  command disable
+
+  # Event is issued when Node process is finished and waiting for debugger to disconnect.
+  # When domain is enabled client should listen for this event instead of `Runtime.executionContextDestroyed`
+  # to check wether node process is finished.
+  event waitingForDisconnect
```

Comment:
Maybe, call it "mainContextDone" or something similar?

Reply:
I'd like to keep as is. This notification should not mention main context since the main purpose of it is provide an idea when node process is not finished only because there is active debugger session.
Runtime.executionContextDestroyed will be send if main context is done.